components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -712,7 +716,12 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                 break;
         }
         return (
-            <StartPage phase={phase} error={error} title={title} showLatestIdeWarning={!isTimedOut && useLatest}>
+            <StartPage
+                phase={phase}
+                error={error}
+                title={title}
+                showLatestIdeWarning={isError || (!isTimedOut && !isStoppingOrStopped && useLatest)}
```

Comment:
```suggestion
                showLatestIdeWarning={useLatest && (isError || (!isTimeout && !isStoppingOrStopped))}
```

But it would be good to name `isTimeout` and `isStoppingOrStopped` together (I can find a proper name for it XD).

Reply:
@mustard-mh Do you mean we should combine both the variables `isTimeout` and `isStoppingOrStopped`?

Reply:
@mustard-mh Could you help or confirm if what @Devansu-Yadav is suggesting above is ok? 🏓

Reply:
> mustard-mh Could you help or confirm if what Devansu-Yadav is suggesting above is ok? 🏓

From the code, yes 👍

Reply:
Thanks, @mustard-mh! Let me loop in @gitpod-io/engineering-webapp to bring this up to the team attention in case this could be ready to merge.